docs/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,676 @@
+- [Multi-tenancy deployment](#multi-tenancy-deployment)
+- [AT-TLS Troubleshooting](#at-tls-troubleshooting)
+  - [The message `This combination of port requires SSL` is thrown when accesing an API ML service through a Browser](#the-message-this-combination-of-port-requires-ssl-is-thrown-when-accesing-an-api-ml-service-through-a-browser)
+  - [AT-TLS rules are not applied](#at-tls-rules-are-not-applied)
+  - [Non matching ciphers / protocols](#non-matching-ciphers--protocols)
+  - [Additional troubleshooting](#additional-troubleshooting)
+- [Full example of AT-TLS configuration](#full-example-of-at-tls-configuration)
+- [Additional Zowe feature configuration with AT-TLS](#additional-zowe-feature-configuration-with-at-tls)
+
+## AT-TLS configuration for Zowe
+
+Follow these steps to configure Zowe to support AT-TLS:
 
-To enable AT-TLS for Zowe components, configure the following parameters:
 ```yaml
 zowe:
     network:
+        # For inbound traffic rules:
         server:
             tls:
                 attls: true
+        # If outbound traffic rules will be configured:
+        client:
+          tls:
+            attls: true
 ```
 
-## Component-Specific Configuration
+While the Zowe Server components do not handle TLS on its own with AT-TLS enabled, the API Mediation Layer (API ML) requires information about the server certificate that is defined in the AT-TLS rule. Ensure that the server certificates provided by the AT-TLS layer are trusted in the configured Zowe keyring. We strongly recommend that AT-TLS be configured with the same Zowe keyring.
+
+:::note Notes
+
+- As the API ML Gateway is a core component of API ML, other components that need to interact with the Gateway, such as Zowe ZLUX App Server, also require AT-TLS configuration.
+
+- Do not set `attls: true` together with `minTls` or `maxTls`. Zowe does not handle TLS in AT-TLS aware mode.
+
+:::
+
+:::caution Important security consideration
+
+Configuring AT-TLS for Zowe requires careful consideration of security settings. These security settings apply to the Client Certificate authentication feature in Zowe API Mediation Layer components, as well as for onboarded services that support the x.509 client certificates authentication scheme.
+
+Outbound AT-TLS rules (i.e. to make a transparent https call through http) that are configured to send the server certificate should be limited to the services that __require__ service to service authentication. If an API ML-onboarded southbound service needs to support X.509 client certificate authentication, we recommend to use the integrated TLS handshake capabilities of API ML. Do not configure an outbound AT-TLS rule for these services.
+
+The Discovery Service endpoints are not reachable by standard API Gateway routing by default.
+
+Zowe v3 includes a new component named ZAAS (Zowe Authentication and Authorization Service). In AT-TLS-aware mode, calls to this service must not include X.509 Client Certificate.
```

Comment:
It must

Reply:
Thanks, copy paste issue from Discovery statement. Regarding the inperson discussion, we are already recommending using only one keyring across the settings.